test/function/decimal_functions_test.cpp

```diff
@@ -47,6 +47,59 @@ TEST_F(DecimalFunctionsTests, SqrtTest) {
   EXPECT_TRUE(result.IsNull());
 }
 
+TEST_F(DecimalFunctionsTests, CeilTestDouble) {
+  std::vector<double> doubleTestInputs = {-36.0, -35.222, -0.7, -0.5, -0.2,
+                                          0.0, 0.2, 0.5, 0.7, 35.2, 36.0,
+                                          37.2222};
+  std::vector<type::Value> args;
+  for (double in: doubleTestInputs) {
+    args = {type::ValueFactory::GetDecimalValue(in)};
+    auto result = function::DecimalFunctions::_Ceil(args);
+    EXPECT_FALSE(result.IsNull());
+    EXPECT_EQ(ceil(in), result.GetAs<double>());
+  }
+
+  args = {type::ValueFactory::GetNullValueByType(type::TypeId::DECIMAL)};
+  auto result = function::DecimalFunctions::_Ceil(args);
+  EXPECT_TRUE(result.IsNull());
+}
+
+TEST_F(DecimalFunctionsTests, CeilTestInt) {
```

Comment:
I put my integer test functions for Ceiling under decimal_functions_test.cpp, should they be moved elsewhere?

Reply:
Sure, that's fine.